### DISCUSSION TITLE
Indent character

### DISCUSSION THREAD

**Discussion by kasfil** [folke/snacks.nvim#314] [Q&A] [answered]
Hi, Did anyone trying to change default indent char? Is it work?
I change the default char to the left aligned line, but it render the default one
![image](https://github.com/user-attachments/assets/7e9542cd-d5b2-49cd-9f44-959abb700089)

Let me know if i made a mistake. thanks

**Comment by baggiponte** [comment]
It happened to me as well. Wasn't sure if I should open up a new issue (s/o @folke).

**Comment by folke** [comment, marked as answer]
Should be `opts.indent.indent.char`.
The first `indent` is the snacks plugin, second `indent` are the options for indentation for the indent plugin.

**Reply by kasfil** [reply]
Sorry, I thought it was on root indent config.
Thank you.
